Task_2/Car.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task_2
+{
+    public class Car
+    {
+        public string Name { get; private set; }
```

Comment:
Since these properties are changed in constructor only, you can skip private setter at all. Leave getter only.